Assets/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PlayerController : MonoBehaviour {
+
+  public float Speed;
+
+  private Rigidbody rigidbody;
+
+  void Start () 
+  {
+    rigidbody = GetComponent<Rigidbody> ();
+  }
+
+  void FixedUpdate () 
+  {
+    float moveHorizontal = Input.GetAxis ("Horizontal");
+    float moveVertical = Input.GetAxis ("Vertical");
+
+    Vector3 movement = new Vector3 (moveHorizontal,0.0f,moveVertical);
+
+    rigidbody.AddForce (movement * Speed);
```

Comment:
[MUST]
If movement means direction, it need to be normalized,  otherwise the norm of vector will not be same.